flows/runs/run.go

```diff
@@ -68,14 +68,8 @@ func (e *runEnvironment) refreshLanguagesCache() {
 type flowRun struct {
 	uuid flows.RunUUID
 
-	contact     *flows.Contact
-	contactUUID flows.ContactUUID
```

Comment:
Runs no longer need to store uuids for flows and contacts because they're loaded after those things are loaded